frontend/src/views/Lobby.tsx

```diff
@@ -180,8 +213,23 @@ function LobbyPage() {
           ))
         }
       </div>
+
+      <MediumText>Difficulty Settings</MediumText>
+      {Object.keys(Difficulty).map((key) => (
+        <DifficultyButton
+          onClick={() => updateDifficultySetting(key)}
+          active={difficulty === Difficulty[key as keyof typeof Difficulty]}
+          enabled={currentUser?.nickname === host?.nickname}
+          title={currentUser?.nickname !== host?.nickname
+            ? 'Only the host can change these settings' : undefined}
```

Comment:
I will note that this `title` occasionally takes a while to show up on hover for me; however, this is inherent to the parameter setting and not something that you can change without choosing a different design, so I think this is totally OK.